streaming/src/queue/queue.h

```diff
@@ -90,10 +90,10 @@ class Queue {
   inline size_t Count() { return buffer_queue_.size(); }
 
   /// Return item count in pending state.
-  size_t PendingCount();
+  inline size_t PendingCount();
```

Comment:
We might define and fulfill all implementation in header file if inline used here.